modules/docker/Makefile

```diff
@@ -0,0 +1,5 @@
+DOCKER ?= docker
+
+DOCKER_RUN ?= "$(DOCKER)" run -i --rm --network=$(DOCKER_NETWORK)
+DOCKER_EXEC ?= "$(DOCKER)" exec -it
+CONTAINER_SHELL ?= sh
```

Comment:
What is the intention of these? I don't see them referenced anywhere (other than `$(DOCKER)`.

I'm not sure why we'd want:
* `docker run -i` without `-t`
  - usuall `-it` go together. In my experience you need to use `-t` for interactive mode
* require setting a network for `docker run` commands 
* `docker run -i`

Reply:
I was playing around with different targets using these, such as a shortcut target for exec (which can get fairly verbose). But I couldn't get the flexibility I wanted, so I ultimately abandoned it. I left these here in case they might be useful later, but I'm happy to remove them.

As far as `docker run -i`, the original purpose of this was to run one-off commands within Docker containers. `-t` allocates a pseudo-TTY, which is necessary when running an interactive shell, but not when you're running a command that is intended to exit after finishing. More background here: https://www.baeldung.com/linux/docker-run-interactive-tty-options

Reply:
> `-t` allocates a pseudo-TTY, which is necessary when running an interactive shell, but not when you're running a command that is intended to exit after finishing.

Right, but `-i` is for "interactive" mode. I get that you might run a single command with `docker run`. I was just confused that you might want the `-i`/`-interactive` (Keep STDIN open even if not attached) option without also using the `-t` option.

I'm not familiar with a case for using one flag without the other, so I thought it might be a mistake.

You can leave them in if you want. I just thought they looked odd.